src/components/shared/Task.tsx

```diff
@@ -77,6 +78,90 @@ function Task(prop: {
           setFileSystem={setFileSystem}
           setCurrentWorkingDirectory={setCurrentWorkingDirectory}
         />
+        {/* TODO: Fix styling for ice-glare-left if displaying the file system render */}
```

Comment:
Minor: Could we extract this svg into its own component for reusability / readability?